src/main/java/org/jitsi/videobridge/AbstractEndpointMessageTransport.java

```diff
@@ -97,14 +96,15 @@ protected void notifyTransportChannelConnected()
      * channel (e.g. file transfer), such that it may interfere with other
      * jitsi messages.
      */
-    @SuppressWarnings("unchecked")
     @Override
     public BridgeChannelMessage endpointMessage(EndpointMessage message)
     {
         String to = message.getTo();
 
         // First insert/overwrite the "from" to prevent spoofing.
-        message.setFrom(getId());
+        String from = getId(message.getFrom());
```

Comment:
side note: having the default behavior of `getId` be specific to `EndpointMessageTransport` really tripped me up here, I think we should make it abstract in this class and do the impls in `OctoEndpointMessageTransport` and `EndpointMessageTransport`, but not an issue for this PR.

Reply:
Haha!  Clearly you felt the same, based on the next commit :)

Reply:
Yeah, I thought about it before, but decided against because I didn't want to duplicate code. And it bit me.